joblib/memory.py

```diff
@@ -882,7 +882,10 @@ class Memory(Logger):
             as functions are evaluated.
 
         bytes_limit: int, optional
-            Limit in bytes of the size of the cache.
+            Limit in bytes of the size of the cache. By default, the size of
+            the cache is unlimited.
+            Note: You need to call :meth:`joblib.Memory.reduce_size` to
```

Comment:
Adding a line break + some bold shoud do the trick?
```suggestion
            the cache is unlimited.  
            **Note:** You need to call :meth:`joblib.Memory.reduce_size` to
```

Reply:
The CI is green I am not touching anything any more 😉. I don't want to see this weird OSX errors.